.gitignore

```diff
@@ -7,7 +7,7 @@
 
 /binaries
 /rulesets
-/src/__tests__/__fixtures__/oas-functions.json
+/__karma__/__fixtures__/
```

Comment:
Doesn't this require a `git rm -r --cached .` to make sure to remove the old files from the index?